tardis/io/util.py

```diff
@@ -13,7 +13,7 @@
 import yaml
 from tqdm.auto import tqdm
 
-from tardis import constants
+from tardis import constants as const
```

Comment:
This change isn't strictly necessary to fix the underlying issue. Nevertheless, it should be fine to keep it.